poetry/console/commands/remove.py

```diff
@@ -73,6 +74,8 @@ def handle(self) -> int:
         self._installer.verbose(self._io.is_verbose())
         self._installer.update(True)
         self._installer.whitelist(requirements)
+        if self.option("lock"):
+            self._installer.lock()
 
         status = self._installer.run()
```

Comment:
Intaller should not run if its a lock only op.

Reply:
Why does the add command then do the same? I wouldn't know what to do in this situation which is why I asked for help on Discord.